HPXMLtoOpenStudio/resources/hvac_sizing.rb

```diff
@@ -691,23 +753,22 @@ def self.process_load_windows_skylights(mj, bldg_design_loads, all_space_design_
         if hr.nil?
           # Average Load Procedure (ALP) load
           clg_htm = htm
-          bldg_design_loads.Cool_Skylights += clg_loads
+          all_zone_loads[zone].Cool_Skylights += clg_loads
           skylight.additional_properties.formj1_values = FormJ1Values.new(area: skylight.area,
                                                                           heat_htm: htg_htm,
                                                                           cool_htm: clg_htm,
                                                                           heat_load: htg_loads,
                                                                           cool_load_sens: clg_loads,
                                                                           cool_load_lat: 0)
-          if fenestration_load_procedure == HPXML::SpaceFenestrationLoadProcedureStandard
-            space_design_loads.Cool_Skylights += clg_loads
+          if space.fenestration_load_procedure == HPXML::SpaceFenestrationLoadProcedureStandard
```

Comment:
MJ8 defines a block load as "the instantaneous load for all rooms served by the central equipment." Since each HVAC system serves a zone (with child spaces), an MJ8 block == an HPXML zone, and so we should always use the standard method for HPXML zones.

And if my understanding is wrong, we can always add more flexibility in the future if someone requests it.